packages/devtools_app/lib/src/debugger/file_search.dart

```diff
@@ -92,6 +93,7 @@ class _FileSearchFieldState extends State<FileSearchField>
   void _onSelection(String scriptUri) {
     final scriptRef = _scriptsCache[scriptUri];
     widget.controller.showScriptLocation(ScriptLocation(scriptRef));
+    programExplorerController.selectScriptNode(scriptRef);
```

Comment:
do we always want to call `programExplorerController.selectScriptNode` when `showScriptLocation` is called? if so, maybe we should put `programExplorerController.selectScriptNode` directly in `showScriptLocation`

Reply:
Good point, that seems to work. Done.